platforms/core-runtime/launcher/src/main/resources/release-features.txt

```diff
@@ -0,0 +1,3 @@
+ - Running Gradle on Java 23
+ - Configuration cache improvements
+ - Build authoring improvements
```

Comment:
I wonder if we could be a little more specific here.

WDYT?
```suggestion
 - Support for Java 23
 - Faster configuration cache 
 - Better configuration cache reports
```

And just drop "build authoring improvements" since there's only one and most users aren't impacted.